app/grandchallenge/blogs/templates/blogs/post_detail.html

```diff
@@ -14,20 +14,36 @@
 
 {% block content %}
     <div class="row mb-3">
-        <div class="col-lg-8">
+        <div class="col-lg-9">
```

Comment:
Be careful as it becomes difficult for users to read blocks of text that are really wide.

Reply:
ok, good point, I can change it back to what it was before ;-)